### PR TITLE
Create resources and building

### LINKED ISSUE #87: Article: Libraries / Frameworks

### PR DESCRIPTION
**Pull request by cefoo**
# Description
Draft for libraries-frameworks.md
Draft for publications.md

In Libraries-frameworks, I created a big table following the example from ModelFront.

In Publications, I left out certain publications which I could not find for free, ie. Early years in Machine Translation, by Hutchins. (Maybe I can mention it without the link?).

These are first drafts.

Fixes #87 

## Type of PR

- Creates the articles:
   - libraries-frameworks.md
   - publications.md

### Checklist:

- [x] I have read the [contributing guidelines](contributing.md).
- [x] I have followed the [style guide](http://machinetranslate.org/style).
- [x] I have made sure that the URL is not already in use.
- [x] I have cross-linked relevant articles.
- [x] I have used the UK spelling.
- [x] I have kept consistency with the structure of sister articles in the same directory.

### CODE REVIEW

File: resources/publications.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+parent: resources
+title: Publications
+description: Scientific publications about machine translation
+---
+
+### Neural machine translation
+
+|     |     |     |
+| --- | --- | --- |
+| 2017 | [Neural Machine Translation](https://arxiv.org/pdf/1709.07809.pdf) | Philipp Koehn |
```
Comment (bittlingmayer) [inline resources/publications.md:11]: I think, per our style guide, paper names should be in Italics

File: resources/publications.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+| 2007 | [Hierarchical Phrase-Based Translation](https://aclanthology.org/J07-2003.pdf) | David Chiang |
+| 2003 | [Statistical machine translation](https://aclanthology.org/N03-1017.pdf) | Philipp Koehn, Franz Josef Och, Daniel Marcu |
+| 2003 | [Statistical Phrase-Based Translation](https://aclanthology.org/N03-1017.pdf) | Philipp Koehn, Franz Josef Och, Daniel Marcu |
+
+
+### Metrics
+
+|     |     |     |
+| --- | --- | --- |
+| 2020 | [COMET: A Neural Framework for MT Evaluation](https://aclanthology.org/2020.emnlp-main.213.pdf) | Ricardo Rei, Craig Stewart, Ana C Farinha, Alon Lavie |
```
Comment (bittlingmayer) [inline resources/publications.md:27]: C.

File: resources/publications.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+| 2016 | [Google’s Multilingual Neural Machine Translation System: Enabling Zero-Shot Translation](https://aclanthology.org/Q17-1024.pdf) | Melvin Johnson, Mike Schuster, Quoc V. Le, Maxim Krikun, Yonghui Wu, Zhifeng Chen, Nikhil Thorat, Fernanda Viégas, Martin Wattenberg, Greg Corrado, Macduff Hughes, Jeffrey Dean |
+| 2016 | [Improving Neural Machine Translation Models with Monolingual Data](https://aclanthology.org/P16-1009.pdf) | Rico Sennrich, Barry Haddow, Alexandra Birch |
+| 2012 | [On the difficulty of training Recurrent Neural Networks](https://arxiv.org/pdf/1211.5063.pdf) | Razvan Pascanu, Tomas Mikolov, Yoshua Bengio |
+
+
+### Open vocabulary
+
+|     |     |     |     |
+| --- | --- | --- | --- |
+| 2016 | [Neural Machine Translation of Rare Words with Subword Units](https://arxiv.org/pdf/1508.07909.pdf) | Rico Sennrich, Barry Haddow, Alexandra Birch |
```
Comment (bittlingmayer) [inline resources/publications.md:85]: I like this style with no "and", maybe we should use it in other places.

File: resources/resources.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+nav_order: 121
+has_children: true
+title: Resources
+description: Machine translation resources
+---
+
+### Calls for papers
```
Comment (bittlingmayer) [inline resources/resources.md:8]: Maybe make this its own article?

Then the auto-generated TOC here will list it.

File: resources/resources.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+| Publication | Organisers | Deadline | Type |
```
Comment (bittlingmayer) [inline resources/resources.md:10]: I think we should def have https://amtaweb.org/amta-2022-1st-call-for-papers-presentations-and-workshop-and-tutorial-proposals/.

We can also add WMT22 already, even if it's just "June 2022"

File: resources/resources.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+| --- | --- | --- | --- |
+| [The impact of Machine Translation in the Audiovisual Translation environment](https://lans-tts.uantwerpen.be/index.php/LANS-TTS/announcement/view/21) | Universitat Jaume I | April 01, 2022 | Scientific publication |
```
Comment (bittlingmayer) [inline resources/resources.md:12]: I thought we were doing dates without the leading *0*?  And maybe without the comma?

File: resources/resources.md (hunk 5 above)
Comment (bittlingmayer) [inline resources/resources.md:10]: If we make the publications italics (which we should), and leave the conferences without italics (which is fine, I guess?), then we can remove the "type" column and save lots of space. :-)

File: resources/resources.md (hunk 6 above)
Comment (cefoo) [reply resources/resources.md:12]: Actually, we agreed on having the leading 0. Every event has the leading 0 as well.

**Comment by bittlingmayer** [comment]
Can we just make an article papers/early-years-in-machine-translation.md?

File: resources/resources.md (hunk 5 above)
Comment (zouharvi) [reply resources/resources.md]: @bittlingmayer Are we really about to make a contract with ourselves to update this page periodically? There are lots of workshops and conferences and keeping track of them all may be problematic.

**Pull request merged**